src/modules/resource/inventory.c

```diff
@@ -470,6 +470,24 @@ static int get_from_kvs (struct inventory *inv, const char *key)
     return rc;
 }
 
+/* Return true if broker.quorum is set to a subset of the full instance.
+ */
+static bool quorum_subset (flux_t *h)
+{
+    const char *quorum;
+    uint32_t size;
+    char buf[32] = "0";
+
+    if (!(quorum = flux_attr_get (h, "broker.quorum"))
+        || flux_get_size (h, &size) < 0)
+        return false;
+    if (size > 1)
+        snprintf (buf, sizeof (buf), "0-%d", (int)size - 1);
```

Comment:
Super minor nit: check the return code for an error?

Reply:
Its not really possible to overflow the buffer unless there were a programming error here, but maybe this function ought to have a chance to fail instead of just returning the logical results.  Let me fix that.

Reply:
I don't even know if I actually check the return code. But I thought I would mention in case you folks do that in `flux-core`. BTW, you do also guard some of the low level system functions from returning prematurely when being attached to a debugger?